### PR TITLE
moved options documentation to separate page

### PR DESCRIPTION
**Pull request by sseraj**
## Purpose
The options table is currently buried in the Plot3d tutorial. I moved the table to a separate page on the top level to increase visibility. This is consistent with the ADflow and IDWarp docs. I also added the text wrapping fix for the table.

## Type of change
- Documentation update

### CODE REVIEW

**Comment by nwu63** [comment]
Currently, all the examples show up on a single page, despite CGNS and Plot3D being listed as separate pages on the side bar. Would you mind splitting it up into separate files so that they will render as separate pages? I think that's preferable to having a top-level heading called `Example Usage` or something and putting everything under that.

**Review by nwu63 — DISMISSED** [review]

File: doc/conf.py [hunk 1]
```diff
@@ -62,3 +62,9 @@
 # relative to this directory. They are copied after the builtin static files,
 # so a file named "default.css" will overwrite the builtin "default.css".
 html_static_path = ["_static"]
+
+html_context = {
+    'css_files': [
+        '_static/theme_overrides.css',  # override wide tables in RTD theme
```
Comment (eirikurj) [inline doc/conf.py:68]: Please change to double quotes to make the file consistent

File: doc/options.rst [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+Here are the options currently available in pyHyp.
+
+===================  ========  =======================================================================================
+Parameter              Type      Description
+===================  ========  =======================================================================================
+`inputFile`           `char`   Name of the file that contains the surface mesh. Here we have a 3D problem and we are
+                               using the surface mesh `uneven_sphere.fmt`. This is a plot3d file which has
+                               been generated in an external meshing program, typically ICEMCFD.
+
+`fileType`            `char`   Type of the input file. Use either 'Plot3d' or 'CGNS'.
```
Comment (eirikurj) [inline doc/options.rst:15]: This is one example of incorrect quotes. Please change to backticks.

**Review by eirikurj — CHANGES_REQUESTED** [review]
Good update.
@sseraj since you are updating these files, can you fix the quotation/backticks issues?
In many places there is a mix of quotes i.e., `` ` `` and `'` resulting in many words not rendered.

**Comment by sseraj** [comment]
I think this addresses the quote/backtick issues.

**Comment by nwu63** [comment]
So technically all the single backticks should be double backticks, since we want the fixed width formatting for those. Would you mind doing that?

**Comment by sseraj** [comment]
The few extra characters wreaked havoc on the table formatting so I had to change the spacing. I split each sentence into its own line like we try to do with the docs.

**Comment by sseraj** [comment]
Travis is failing on `py3-centos7-impi` because: `Error: Product support for your (Comp-FL) license has expired.`

**Comment by nwu63** [comment]
Thanks for that, table looks much more legible now. Yeah I'm not sure why the license expired, I thought we build that nightly. Anyways, I'm fine merging this in since it only touches the docs.

**Review by nwu63 — APPROVED** [review]

**Review by eirikurj — APPROVED** [review]
I am approving for now, but we need to adjust many of the other pages in this repo as well as its mixed and single backticks all over the place.
We should adjust how we present the options table for other repos also.

**Comment by nwu63** [comment]
Yeah, now that we are rolling out the CSS "hack" for tables, I am much more inclined to throw that in a repo and upload it to pyPI. Will comment on the appropriate issue and elicit help.

**Pull request merged**